src/MasterMix/types.ts

```diff
@@ -0,0 +1,36 @@
+import { Modify } from '@mll-lab/js-utils';
+import { ColumnType } from 'antd/es/table';
+import { ReactNode } from 'react';
+
+export type Ingredient = {
```

Comment:
```suggestion
export type MasterMixIngredient = {
```

If we export it to package users